health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationWrapper.js

```
@@ -693,6 +786,29 @@
+      <Footer
+        actionFields={[
+          <Button
+            type={"button"}
+            style={{ marginLeft: "2.5rem", width: "14rem" }}
+            label={t("HCM_BACK")}
+            variation={"secondary"}
+            t={t}
+            onClick={() => {
+              window.history.back();
+            }}
+          ></Button>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)